osx/Fuse/README.md

```diff
@@ -4,24 +4,42 @@ KBFuse is OSXFuse (or Fuse for MacOS) "branded" for Keybase. This is so we can m
 conflicting with existing OSXFuse installs. It also allows us to sign the kext with our certificate instead
 of relying on 3rd party binaries shipped from other developers.
 
+For simplicity, we'll drop 10.10 and only support 10.11+ for the fuse
+extension. See the "Notes on 10.10" below for how to build for 10.10.
+
 ### Pre-requisites
 
-#### Build from Xcode 7
+#### Xcode
+
+First of all, we can't have a kernele extension built for a newer kernel and
+run in on older kernels, but we can generally have a kernel extension built for
+an older kernel and run it on newer kernels. So we'll try to build for the
+oldest possible (compatible) kernel, to have a fewer kernel extensions built as
+possible.
 
-In order to support 10.10 and above, you'll need to build from an older Xcode
-version.
+For OSXFuse, it's possible to use a kernel extension that's built against the
+macOS 10.11 SDK on macOS 10.12, 10.13, and 10.14. So we'll build for 10.11
+kernel, then we'd make symlinks for 10.12, 10.13, and 10.14 whose target is
+10.11. All these are handled by the build script, but we'll need the
+appropriate SDKs to build against.
 
-- Download Xcode 7 from /keybase/public/gabrielh/Xcode7.app.zip
-- Place in /Applications (as Xcode7.app)
-- `sudo xcode-select --switch /Applications/Xcode7.app`
-- Move /Applications/Xcode.app out of the way: `sudo mv /Applications/Xcode.app /tmp`
+Xcode 7.3.1 is the latest version that includes the macOS 10.11 SDK, so we'll
+need it installed on the macOS where you build the KBFuse bundle.
+he macOS where you build the KBFuse bundle.
 
-Afterwards, you should move Xcode.app back to /Applications:
+Older versions of the Xcode can be downloaded from [Apple official developer
+site](https://developer.apple.com/download/more/). It has to live under
+`/Applications`, but under a different name, e.g. `/Applications/Xcode-7.3.1`.
+Don't worry about making the naming scheme exact, as the OSXFuse builder is
+able to spotlight different versions automatically.
+[Here](https://medium.com/@hacknicity/working-with-multiple-versions-of-xcode-e331c01aa6bc)'s
+a blog post that explains the process of having multiple Xcode installed on the
+same machine.
 
-- `sudo mv /tmp/Xcode.app /Applications/`
-- `sudo xcode-select --switch /Applications/Xcode.app`
+#### Signing certificate
 
-You'll also need the Keybase signing certificate from someone at Keybase.
+You'll also need the Keybase signing certificate. Ask around if you don't have
+access to it.
```

Comment:
Note that we are dropping macOS 10.10 support in KBFuse. On the other hand I'm also not sure if it used to work on 10.10.